features/step_definitions/confirmation_steps.rb

```diff
@@ -1,14 +1,17 @@
-Given(/^I have a unconfirmed account that is over (\d+) days old$/) do |arg1|
-  @user = FactoryGirl.create(:user, confirmed_at: nil, created_at: 2.days.ago)
-  @user.update_attributes(confirmation_sent_at: 2.days.ago)
+Given(/^I have a unconfirmed account that is over (\d+) days old$/) do |numdays|
+  @user = FactoryGirl.create(:user,
```

Comment:
I understand rubocop is saying this is a 80 character line violation. Sometimes its better to have a line over 80 characters though if it makes it easier to read. I think this is one of those cases, this code is very hard to read when it's like this.